src/guiapi/src/window_menu.cpp

```diff
@@ -77,47 +80,111 @@ IWindowMenuItem *window_menu_t::GetActiveItem() {
     return GetItem(index);
 }
 
+bool window_menu_t::moveToNextVisibleItem(int steps) {
```

Comment:
My personal opinion is: 3 methods would be better readable
`bool window_menu_t::moveToNextVisibleItem() ` move +1 step
`bool window_menu_t::moveToPrevVisibleItem() ` move -1 step
`bool window_menu_t::moveToVisibleItem(int steps) ` call Next and Prev methods inside this one